example/detox/e2e/test/client/create_api_client.e2e.js

```diff
@@ -32,9 +37,73 @@ describe("Create API Client", () => {
         exponentialBackoffScale: `${getRandomInt(5) + 3}`,
         retryInterval: `${getRandomInt(5) + 4}`,
     };
-    const { clientListScrollView, removeClientWithName } = ClientListScreen;
+
```

Comment:
Now that there are many kinds of servers. Would it make sense to add service requirement per spec file for ease of troubleshooting? Something like `await apiRequire[ServerType]`?

Reply:
See comment https://github.com/mattermost/react-native-network-client/pull/57#issuecomment-815842308